css/admin.block-post.css

```diff
@@ -1,6 +1,7 @@
 #minor-publishing-actions,
 #misc-publishing-actions #visibility,
-#misc-publishing-actions .curtime {
+#misc-publishing-actions .curtime,
```

Comment:
Is hiding a UI action component by CSS a good idea/best practices?
_actually reviews code_

Reply:
Unfortunately, it's the only way. The other option would be to create our own version of the meta box, which would be a mess of duplicate code (and ongoing support).